chance/p5js/chance-contrast/index.md

```diff
@@ -0,0 +1,9 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/
+title: Law of Casuality
+tags: [P5.js, Law of Causality, Law of Connected-Elements, Law of Relativity of Properties, Form, Grid, Interaction, Repeat]
+permalink: /Chance/Contrast/
```

Comment:
permalinks are lowercase. the per,alink should contain the title of the sketch. Contrast is to broad. If possible use the name of the folder.   

That means title, permalink and sketch folder should be the same.